### PR TITLE
First issue template for PDF Arranger

### PR DESCRIPTION
**Pull request by jeromerobert**

### CODE REVIEW

**Comment by dreua** [comment]
In #447 I said I'd take care of that, but didn't come around to it yet. Thanks for taking the initiative!

Already looking good to me, I think we can merge this in any state and refine it as needed.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
```
Comment (dreua) [inline .github/ISSUE_TEMPLATE/bug_report.md:10]: After finding the "rich diff" button: I think there needs to be a second line break after each `**heading**`. You could also consider using `### heading` (or some other level) instead.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+A clear and concise description of what you expected to happen.
+
+**PDF files**
+Check if the bug appear only with some PDF. If the bug is specific to some PDF files please attach one to the issue.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**System and Versions**
+ - PDF-Arranger version and where you get it (distro package, flatpak, pip, git, download, ...)
```
Comment (dreua) [inline .github/ISSUE_TEMPLATE/bug_report.md:30]: I'd prefer "PDF Arranger" to be consistent. "got" could be used as past tense of "get" but the message is clear anyway.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+ - pikepdf version and where you get it (distro package, pip, git, ...)
+ - OS name and version (ex: Windows 10, Fedora 33, ...)
```
Comment (dreua) [inline .github/ISSUE_TEMPLATE/bug_report.md:32]: Are you using Fedora, too? Did I convince you with my constant advertising? :laughing:

Comment (dreua) [reply .github/ISSUE_TEMPLATE/bug_report.md:32]: I'd prefer `e.g.` over `ex:` but it's both alright with me. Picking details here, sorry.

Comment (jeromerobert) [reply .github/ISSUE_TEMPLATE/bug_report.md:32]: > Are you using Fedora, too? Did I convince you with my constant advertising?

Oh no, when I'm buried I'll still use Debian, but I knew you'd like that ;-)

**Comment by angsch** [comment]
> **PDF files**
Check if the bug appear only with some PDF. If the bug is specific to some PDF files please attach one to the issue.

Having img2pdf in mind, what about generalizing _PDF files_ to _documents_?

**Pull request merged**